platform/node/index.js

```diff
@@ -19,7 +19,12 @@ var Map = function(options) {
     return new constructor(Object.assign(options, {
         request: function(req) {
             request(req, function() {
-                req.respond.apply(req, arguments);
+                var args = arguments;
+                // Protect ourselves from `request` implementations that try to release Zalgo.
```

Comment:
Sorry, that was overly snarky. I've rephrased the comment, and kept the link for further background.